site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -49,6 +49,31 @@ extensions][malicious-mac]).
 
 ## Before you begin {: #prereqs }
 
+### Creating a CRX file {: #create-crx }
+
+Some of the installation methods described here require a CRX file. The procedure below shows how to do this. It requires a private key file, which you can create with OpenSLL or any other method of creating a key file. If you do not have one, Chrome will create one for you.
```

Comment:
The key is only required when upgrading to a new version.

Reply:
When the developer releases a new version of their extension by increasing the [version](https://developer.chrome.com/docs/extensions/mv3/manifest/version/) key in the manifest.

Reply:
I'm missing something. The dialog box used here doesn't know whether I've incremented the version number in my manifest and it has no way to track that. In what specific circumstances must I increment the version and supply a new key? What system enforces that?